File: firstcode/firstassignemt/myjavaassignment/myfunctions/MyDiscount.java
import java.util.Scanner;

public class MyDiscount {
	public static void main(String[] args) {	
		System.out.println("Welcome friend, today is just so beautiful\n");
	
	Scanner input =  new Scanner(System.in);
		int price;
		int percentageDiscounts;
	System.out.println("HILLARY SUPERSTORE & Co.\n");
	System.out.print("Enter the price: ");
		price = input.nextInt();
	System.out.print("Enter the percentage discount: ");
		percentageDiscounts = input.nextInt();

	CommonTools tools = new CommonTools();

	double discount = tools.mydiscount(price, percentageDiscounts);
		
	System.out.printf("Your discount price is %.2f%n", discount);
		
		
}
}

File: firstcode/firstassignemt/newassignement/mydiscount.py
def mydiscount(price, discount):
	percentage = (discount/100)
	
	percentage_of_price = (percentage * price)
	new_discount_price = (price - percentage_of_price)
	
	return new_discount_price


print("WELCOME TO HILLARY SUPERSTORE")

price = float(input("Enter the price of the goods:  "))
discount = float(input("Enter the discount in percentage: "))
new_price_in_discount = mydiscount(price, discount)
print(f"the new price is {new_price_in_discount}")